.travis.yml

```
@@ -0,0 +1,45 @@
+dist: trusty
+
+cache:
+  apt: true
+  directories:
+    # Cache directory for older Composer versions.
+    - $HOME/.composer/cache/files
+    # Cache directory for more recent Composer versions.
+    - $HOME/.cache/composer/files
+
+language:
+  - php
+
+php:
+  - 5.6
+  - 7.0
+  - 7.1
+  - 7.2
+  - 7.3
+  - "7.4snapshot"
+
+matrix:
+  fast_finish: true
+  include:
+    - php: 7.3
+    - php: 7.2
+    - php: 7.1
+    - php: 7.0
+    - php: 5.6
+
+  allow_failures:
+    # Allow failures for unstable builds.
+    - php: "7.4snapshot"
+
+before_install:
+    - composer install --dev
```

```suggestion
    - composer install
```
The Composer default is to install `--dev` nowadays.